aether/sdk/drf/serializers.py

```diff
@@ -87,19 +87,18 @@ def get_url(self, obj, view_name, request, format):
 
 
 # https://www.django-rest-framework.org/api-guide/serializers/#dynamically-modifying-fields
-class DynamicFieldsModelSerializer(serializers.ModelSerializer):
+class DynamicFieldsSerializerMixin(object):
     '''
-    A ModelSerializer that takes two additional arguments ``fields`` and ``omit``
+    Add addtional functionality to Serializers adding two arguments ``fields`` and ``omit``
```

Comment:
```suggestion
    Add additional functionality to Serializers adding two arguments ``fields`` and ``omit``
```

Reply:
I added a spell-check to my editor and I still miss crap like this because it flags things like  `serializers` as misspelled...